addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2068,6 +2068,9 @@ const Wysiwyg = Widget.extend({
         finalOptions.autohideToolbar = typeof finalOptions.autohideToolbar === 'boolean'
             ? finalOptions.autohideToolbar
             : !finalOptions.snippets;
+        if (finalOptions.inlineStyle) {
+            finalOptions.dropImageAsAttachment = false;
```

Comment:
Maybe add a comment about how inlineStyle means that the convert inline function is called and that it already handles the conversion ?